classes/PodsAPI.php

```diff
@@ -1203,6 +1203,18 @@ public function get_wp_object_fields( $object = 'post_type', $pod = null, $refre
 					'type'  => 'slug',
 					'alias' => array( 'nicename', 'slug', 'permalink' )
 				),
+				'first_name'  => array(
```

Comment:
:warning: Array double arrow not aligned correctly; expected 6 space(s) between "'first_name'" and double arrow, but found 2.

<sub>hash: 50dfb9b4b6d8c2349f4275fb5bb9a19f</sub>